frontend/src/AddQuestionPage.js

```diff
@@ -8,6 +17,8 @@ const AddQuestionPage = () => {
     const [translatedQuestions, setTranslatedQuestions] = useState({});
     const [translatedAnswers, setTranslatedAnswers] = useState({});
     const [translatedOthers, setTranslatedOther] = useState({});
+    const [areQuestionsLoading, setQuestionsLoading] = useState(false);
+    const [areAnswerssLoading, setAnswersLoading] = useState(false);
```

Comment:
Think there is a typo here in `areAnswerssLoading`. I would correct this and everywhere it's referenced.